src/constants/app.constants.ts

```diff
@@ -4,45 +4,51 @@ export const defaultBudget: ICategory[] = [
   {
     title: "Food",
     amount: 2000,
-    unmanaged: false,
+    managed: true,
+    editable: false,
```

Comment:
`editable` field is probably no longer required, since we're no longer using a default budget concept.

Also, this field is not present in the `category` type, will cause issue if we use this constant anywhere in the app later.